hiera/hieradata/dev/wso2/wso2as/5.3.0/manager.yaml

```diff
@@ -0,0 +1,34 @@
+---
+wso2::mgt_hostname : mgt.as.wso2.com
+
+wso2::clustering :
+    enable : true
+    local_member_host : 192.168.100.13
```

Comment:
We don't need the static ip for local_member_host. Fixed in commit 674a0f05c13c282768c1f6a31f7e66aa9fbe2018.